### PR TITLE
Add Register Screen 

### PR DESCRIPTION
**Pull request by ghost**
The registration Screen content was accidentally replaced by the Register screen content. This PR reverts that and also creates a registerScreen with the form.

### CODE REVIEW

File: src/screens/RegisterScreen/index.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { ActivityIndicator, AsyncStorage, Image, Text, View } from 'react-native';
+import { NavigationActions } from 'react-navigation';
+import { NavHeader, Screen, Registration} from '../../components';
+import styles from './styles';
+import theme from '../../theme';
+
+export default class RegistrationsScreen extends React.Component {
```
Comment (smitch88) [inline src/screens/RegisterScreen/index.js]: Name change of class here to match the file export

File: src/screens/RegisterScreen/index.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  constructor (props) {
+    super(props);
+    this.state = {
+      error: null,
+      isAuthenticating: false
```
Comment (smitch88) [inline src/screens/RegisterScreen/index.js]: I don't think this key is necessary here perhaps "saving or registering" makes more sense based on how I see it being used.

**Review by smitch88 — CHANGES_REQUESTED** [review]
Couple small changes looks fine other than those. Merge after fixes.

**Review by nick-brady — APPROVED** [review]
lgtm

**Review by smitch88 — APPROVED** [review]

**Pull request merged**